test/metatx/MinimalForwarder.test.js

```diff
@@ -55,8 +61,9 @@ contract('MinimalForwarder', function (accounts) {
           expect(await this.forwarder.getNonce(this.req.from)).to.be.bignumber.equal(web3.utils.toBN(this.req.nonce));
         });
 
-        it('success', async function () {
-          expect(await this.forwarder.verify(this.req, this.sign())).to.be.equal(true);
+        it.only('success', async function () {
```

Comment:
Its ok to change to `it.only` for local tests, but that should never be commited.

@frangio can we add a rule so that CI detects that if we are not carefull?

Reply:
Keeping `.only` will cause coverage checks to fail, and we will notice it drop to like 1%.

Reply:
Solving since the `only` was removed and the coverage check is already there.